api/msearch.go

```diff
@@ -97,7 +97,7 @@ func WithMsearchSourceParam(sourceParam string) MsearchOption {
 // body: the request definitions (metadata-search request definition pairs), separated by newlines.
 //
 // options: optional parameters.
-func (a *API) Msearch(body []interface{}, options ...MsearchOption) (*MsearchResponse, error) {
+func (a *API) Msearch(body map[string]interface{}, options ...MsearchOption) (*MsearchResponse, error) {
```

Comment:
This code is generated by the `generator/api` package, if it needs a fix it needs to go there